### DISCUSSION TITLE
Error while loading shared libraries: libicui18n.so.60

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/wslg#346] [Q&A] [answered]
I'm new to Linux in general and have been trying WSLg on Windows 11 today. I've successfully installed a couple software, but both of them can't run with the following error:

`error while loading shared libraries: libicui18n.so.60: cannot open shared object file: No such file or directory`

I think it's something to do with the Qt library, but I'm unsure how to resolve it on WSLg. I've tried several suggestions I found, but none of them has allowed me to run the apps yet. For context, I've run Gimp just fine.

Is this error specific to WSLg? How can I resolve this?

**Comment by bacongobbler** [comment]
Did you make sure you have ICU installed? Try installing it with `apt install libicu-dev`.

**Reply by myokeeh** [reply]
libicu-dev is already the newest version (66.1-2ubuntu2).

**Reply by myokeeh** [reply]
@bacongobbler any other ideas?

**Comment by myokeeh** [comment]
As of today, Windows 11 (22000.258) and with the new Windows Store WSL install preview, I still get the same error launching this particular app. For reference this is the app I'm trying to launch:

http://downloads.quanergy.com/qview/qview_readme.txt

**Comment by sirredbeard** [comment, marked as answer]
This app is designed for Ubuntu 18.10 bionic.

![image](https://user-images.githubusercontent.com/33820650/158892392-865122bc-5e2f-42e2-b953-cb41e63c58ce.png)

You are attempting to install on Ubuntu 20.04 focal. I was able to reproduce the bug on Ubuntu 20.04 focal, it doesn't work.

The appears to be looking for `libicui18n.so.60` which is not packaged for Ubuntu 20.04 '[focal](https://packages.ubuntu.com/search?suite=focal&arch=any&mode=exactfilename&searchon=contents&keywords=libicui18n.so.60)'. 

libicui18n.so.60 is available in Ubuntu 18.10 '[bionic](https://packages.ubuntu.com/search?suite=bionic&arch=any&mode=exactfilename&searchon=contents&keywords=libicui18n.so.60)'.

You can download Ubuntu 18.10 for WSL [here](https://www.microsoft.com/store/productId/9N9TNGVNDL3Q).

Then install with:

```
sudo apt update
sudo apt upgrade -y
sudo apt install libharfbuzz-dev -y
wget https://downloads.quanergy.com/qview/linux/quanergy-qview_1.6.367_amd64.deb
sudo dpkg -i quanergy-qview_1.6.367_amd64.deb
/opt/quanergy/Q-View/Q-View
```
